### PR TITLE
added metabat

### PR DESCRIPTION
**Pull request by HassanAmr**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/metabat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - boost >=1.55.0,{{CONDA_BOOST}}*
+    - zlib >=1.2.4,{{ CONDA_ZLIB }}*
+
+test:
+  commands:
+    - metabat -h
+
+about:
+  home: https://bitbucket.org/berkeleylab/metabat
+  license: Free for academic, nonprofit and personal use
```
Comment (tomkinsc) [inline recipes/metabat/meta.yaml]: Can you clarify the license? There doesn't appear to be a commercial restriction in the [`license.txt`](https://bitbucket.org/berkeleylab/metabat/raw/235ca312d75583514c20278278522cea291562c8/license.txt) file. The short id to use for the LBL [license](https://spdx.org/licenses/BSD-3-Clause-LBNL.html) is `  license: BSD-3-Clause-LBNL`.

Comment (HassanAmr) [reply recipes/metabat/meta.yaml]: You mean I should ask the author of this tool to take the commercial use into consideration?

**Comment by mdehollander** [comment]
Is this recipe ready to be merged?

**Comment by HassanAmr** [comment]
I am trying on another branch to build from source. If it works, I will discard this branch. Other than that, this is ready.

File: recipes/metabat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  sha256: 1fc70cb388feac43392aa87190cd368797cb88dbd0be8b541b8019187436dbc3
+  patches:
+    - metabat.patch
+
+requirements:
+  run:
+    - python
+    - perl
+    - boost >=1.55.0,{{CONDA_BOOST}}*
+    - zlib >=1.2.4,{{ CONDA_ZLIB }}*
```
Comment (mbargull) [inline recipes/metabat/meta.yaml]: You can revert this to
```yaml
    - boost >=1.55.0
    - zlib >=1.2.4
```
now as changes to the linter were made in the meantime. (Also, expressions like `>=1.55.0,{{CONDA_BOOST}}*` aren't likely what you actually want.)

File: recipes/metabat/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+source:
+  url: https://bitbucket.org/berkeleylab/metabat/downloads/metabat-static-binary-linux-x64_v2.12.1.tar.gz
+  sha256: 1fc70cb388feac43392aa87190cd368797cb88dbd0be8b541b8019187436dbc3
+  patches:
+    - metabat.patch
+
+requirements:
+  run:
+    - python
```
Comment (mbargull) [inline recipes/metabat/meta.yaml]: I think -- but haven't checked that entirely -- that `python` is only a build requirement if you were to build from source.

**Comment by mbargull** [comment]
> I am trying on another branch to build from source. If it works, I will discard this branch.

If you get that to work, that'd be nice, of course.

**Comment by HadrienG** [comment]
Hi! What are we missing here to merge this? I'd like to pick this up.

**Comment by epruesse** [comment]
closing in favor of #15411

**Pull request closed without merging**